apps/admission-webhook-k8s/admission-webhook.yaml

```
@@ -13,6 +13,8 @@ spec:
     metadata:
       labels:
         app: admission-webhook-k8s
+        dns-name: admission-webhook-svc
```

Could we use labels related to   spiffe.io  to avoid confusions?
```suggestion
        spiffe.io/spiffe-dns-webhook:  ...
```

---

Thanks, fixed